features/create_articles.feature

```diff
@@ -0,0 +1,13 @@
+Feature: Create articles
+  As a journalist
+  In order to attract subscribers
+  I would like to be able to create articles
+
+  Background:
+    Given I visit the "index" page
+
+  Scenario: Successfully create an article
+    When I click "New Article" link
+    Then I fill in "Title" with "A Whole New World"
+    And I fill in "Content" with "A new fantastic point of view"
+    And I click "Create Article" button
```

Comment:
And then what?

Reply:
yeah we added a Then test that checks for the user to visit the homepage after successful creation